modules/serverless-logic-restarting-the-build-preview-mode.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-restarting-the-build-preview-mode_{context}"]
+= Restarting a build
+
+To restart a build, you can add or edit the `sonataflow.org/restartBuild: true` annotation in the `SonataFlowBuild` instance. 
+ 
+.Prerequisites
+
+* {ServerlessLogicOperatorName} is installed on your cluster.
+* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in OpenShift Container Platform.
+* You have created your {ServerlessLogicProductName} project
+* You have installed the OpenShift CLI `(oc)`.
+
+.Procedure
+
+. Check if the `SonataFlowBuild` instance exists.
+
+. Edit the `SonataFlowBuild` instance.
++
+[source,terminal]
+----
+$ oc edit sonataflowbuild/<name> -n <namespace>
+----
+
++
+Replace `<name>` with the name of your `SonataFlowBuild` instance and `<namespace>` with the namespace where your workflow is deployed.
+
+. Add the `sonataflow.org/restartBuild: true` annotation to restart the build.
++
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowBuild
+metadata:
+  name: <name>
+  annotations:
+    sonataflow.org/restartBuild: true
+----
++
+This action will trigger the {ServerlessLogicOperatorName} to start a new build of the workflow.
```

Comment:
Is this true (just editing it triggers the build)? Or do thy have to save the file first?

```suggestion
This action triggers the {ServerlessLogicOperatorName} to start a new build of the workflow.
```